scalafmt-cli/src/main/scala/org/scalafmt/cli/Cli.scala

```diff
@@ -96,7 +96,7 @@ object Cli {
     "true" == System.getProperty("scalafmt.native-image", "false")
 
   private def getProposedConfigVersion(options: CliOptions): String =
-    s"version = '$stableVersion'"
+    s"version = $stableVersion"
```

Comment:
could you make the same change on line 137 below (where it was copied from)?

Reply:
Done, thanks for the heads up 👍